project_documentation/Definition_of_done.txt

```diff
@@ -0,0 +1,24 @@
+Definition of Done:
+
+a User Story (US) is considered Done when all of the following are fullfilled:
+* The assigned reviewer(s) has checked that all the Acceptance Criterias are fullfilled, if there are any Acceptance Criterias
+* All the tasks related to the US are Done
+* The reviewer(s) has tested the US functionality in Android studio and not noticed any errors
+* if the US functionality as a whole was branched out, the Pull Request process has been completed 
+
+Currently we have no coding guidelines that guide the reviewer on what to react on,
+Currently we have no requirements on level of unit test, regression tests or manual tests. only requirement on testing is that Aceptance Criteras shall be fullfilled and the US should be resolved.
+
+
+
+Pull Request process:
+When functionality is beeing beeing developed in a separate branch the branch shall be rebased, if main has evolved, and merged.
```

Comment:
"beeing beeing" is a little confusing

Reply:
what do you mean? Its perfect 😃

Reply:
thanks, I have updated in latest version.

now it says
"When functionality is being developed  ..."

is is this issue resolved, @ItsPonks ? if so please press resolve converssation for this issue.